tasks/basic.py

```diff
@@ -18,7 +18,31 @@ def __init__(self, seconds: int, *args, **kwargs):
     def run(self):
         time.sleep(self.seconds)
         return self.seconds
+        
+class SleepUntilTask(Task):
+    def __init__(self, target_date_str: str, *args, **kwargs):
```

Comment:
Is there any particular reason to use string as a type for the target date? You can probably pass a datetime object instead, it will also facilitate further code.

Reply:
The corresponding changes have been implemented